sdks/core/src/main/java/org/gbif/pipelines/core/config/model/VocabularyConfig.java

```diff
@@ -18,14 +18,13 @@ public class VocabularyConfig implements Serializable {
   // directory where the vocabulary files are stored
   private String vocabulariesPath;
 
-  private Map<Term, String> vocabulariesNames = Collections.emptyMap();
+  private Map<String, String> vocabulariesNames = Collections.emptyMap();
```

Comment:
This was because this vocabulary is used for 10 terms so instead of adding the 10 terms to the map I thought to use this as key https://github.com/gbif/dwc-api/blob/c366f91c5cb0cb2eb68d40b9567fe45b9a9ead0a/src/main/java/org/gbif/dwc/terms/DwcTerm.java#L279C30-L279C53 

And then do this check: https://github.com/gbif/pipelines/blob/41ed816fbb3516436805e62472f455df86d56bf7/sdks/core/src/main/java/org/gbif/pipelines/core/parsers/vocabulary/VocabularyService.java#L23

What do you think?